spec/integ/crypto/verification.spec.ts

```diff
@@ -808,6 +810,115 @@ describe.each(Object.entries(CRYPTO_BACKENDS))("verification (%s)", (backend: st
+                    },
+                    user_id: "@bob:xyz",
+                    usage: ["self_signing"],
+                    signatures: {
+                        "@bob:xyz": {
+                            "ed25519:i2y3s50NUzxxtIdyxEc9yFybLIq6qgBSb9r7k9yICBY":
+                                "SrAMqvGSkwzxXoLZAcmuMl+gbQBlHbWObY1IX1wZ9ADlOJkeEzzjxzsyah25I29TLFHrTG+dFphreItE3pGQAA",
+                        },
+                    },
+                },
+            },
+            user_signing_keys: {
+                "@bob:xyz": {
+                    keys: {
+                        "ed25519:mdF9bCRhssc1OTWqsmSdS4usif2PboaT/cAJMNqykbA":
+                            "mdF9bCRhssc1OTWqsmSdS4usif2PboaT/cAJMNqykbA",
+                    },
+                    user_id: "@bob:xyz",
+                    usage: ["user_signing"],
+                    signatures: {
+                        "@bob:xyz": {
+                            "ed25519:i2y3s50NUzxxtIdyxEc9yFybLIq6qgBSb9r7k9yICBY":
+                                "7DoXVLc4iNvQQOSR9G9QQEot93M7CM76z0fUtraIo/hnMH1/M96DIskLl9MBONOL5nEekw2LXXFWh55AANMMCg",
+                        },
+                    },
+                },
+            },
+        };
+
+        const deviceKeys: IDeviceKeys = {
+            algorithms: ["m.olm.v1.curve25519-aes-sha2", "m.megolm.v1.aes-sha2"],
+            device_id: "bob_device",
+            keys: {
+                "curve25519:bob_device": "F4uCNNlcbRvc7CfBz95ZGWBvY1ALniG1J8+6rhVoKS0",
+                "ed25519:bob_device": "PNsWknI+bXfeeV7GFfnuBX3sATD+/riD0gvvv8Ynd5s",
+            },
+            user_id: "@bob:xyz",
+            signatures: {
+                "@bob:xyz": {
+                    "ed25519:bob_device":
+                        "T+UUApQlgZHNfM4gdkcrJcjTmeF7wEYAZH2OS99gKKizEgEFL89acAC7yuLfm2MI5J9Ks2AGkQJp3jjNjC/kBA",
+                },
+            },
+        };
+
+        const bobId = "@bob:xyz";
+
+        beforeEach(async () => {
+            aliceClient = await startTestClient();
+
+            e2eKeyResponder.addCrossSigningData(crossSingingKeys);
+            e2eKeyResponder.addDeviceKeys(deviceKeys);
+
+            syncResponder.sendOrQueueSyncResponse(getSyncResponse([bobId]));
+
+            // Wait for the sync response to be processed
+            await syncPromise(aliceClient);
+        });
+
+        function awaitRoomMessageRequest(): Promise<IContent> {
+            return new Promise((resolve) => {
+                fetchMock.put(
+                    "express:/_matrix/client/v3/rooms/:roomId/send/m.room.message/:txId",
+                    (url: string, options: RequestInit) => {
+                        resolve(JSON.parse(options.body as string));
+                        return { event_id: "$YUwRidLecu:example.com" };
+                    },
+                );
+            });
+        }
+
+        newBackendOnly("alice send verification request in a DM to bob", async () => {
```

Comment:
```suggestion
        newBackendOnly("alice sends a verification request in a DM to bob", async () => {
```

Reply:
doesn't this work on the old backend?

Reply:
The old backend fails because one of the device is not verified. Is it something we want here ?

Reply:
it would certainly give me more confidence in the tests if we could make it work on both stacks.

I'm a bit surprised by the error: isn't the whole *point* that we haven't verified Bob's devices?